sites/all/modules/publizon/includes/publizon.auth.inc

```diff
@@ -32,6 +32,11 @@ function publizon_auth_single_sign_on($name, array $extra) {
   $client = PublizonUserClient::getClient();
   $support_id = $client->getSupportId($name, $retailer_id);
   if ($support_id) {
+    // If only one tag is set at adgangsplatformen this is a string, so convert
+    // to array if that's the case.
+    $tags = $extra['attributes']['userPrivilege'] ?? array();
+    $tags = is_array($tags) ? $tags : array($tags);
```

Comment:
If you know that `$extra['attributes']['userPrivilege']` is an array (llst) or a scalar you can just use `(array)$extra['attributes']['userPrivilege']` to create an array with the scalar as a single element (cf. https://www.php.net/manual/en/language.types.array.php#language.types.array.casting).

Reply:
Nice trick, will update the code to morrow

Reply:
Fixed